package proto_ff;

import "yd_fieldoptions.proto";


message operateoperate
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "活动ID"];
	optional int32 huoDong = 2[(yd_fieldoptions.field_cname) = "所属活动"];
	optional string name = 3[(yd_fieldoptions.field_cname) = "活动名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 nameId = 4[(yd_fieldoptions.field_cname) = "活动细分类型"];
	optional string tiaoJian = 5[(yd_fieldoptions.field_cname) = "达成条件", (yd_fieldoptions.field_bufsize) = 300];
	optional int32 servertype = 6[(yd_fieldoptions.field_cname) = "服务器类型"];
	optional string timeA = 7[(yd_fieldoptions.field_cname) = "开始时间", (yd_fieldoptions.field_bufsize) = 60];
	optional string timeB = 8[(yd_fieldoptions.field_cname) = "结束时间", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 timeType = 9[(yd_fieldoptions.field_cname) = "时间类型"];
	optional string miaoSu = 10[(yd_fieldoptions.field_cname) = "规则描述", (yd_fieldoptions.field_bufsize) = 180];
	optional int32 ifOpen = 11[(yd_fieldoptions.field_cname) = "活动开关"];
}


message Sheet_operateoperate
{
	repeated operateoperate operateoperate_List = 1[(yd_fieldoptions.field_arysize)=40];
}

message operatetutor
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 condition = 3[(yd_fieldoptions.field_cname) = "条件"];
	optional int64 conditionParam1 = 4[(yd_fieldoptions.field_cname) = "条件参数"];
	optional int64 conditionParam2 = 5[(yd_fieldoptions.field_cname) = "条件参数"];
	optional int64 boxId = 6[(yd_fieldoptions.field_cname) = "奖励ID"];
	optional int32 teach = 7[(yd_fieldoptions.field_cname) = "教诲值"];
}


message Sheet_operatetutor
{
	repeated operatetutor operatetutor_List = 1[(yd_fieldoptions.field_arysize)=80];
}

message operatejiangli
{
	optional int64 tiaoJianId = 1[(yd_fieldoptions.field_cname) = "达成条件"];
	optional string huoDong = 2[(yd_fieldoptions.field_cname) = "所属活动名称", (yd_fieldoptions.field_bufsize) = 60];
	optional string downNu = 3[(yd_fieldoptions.field_cname) = "参数一", (yd_fieldoptions.field_bufsize) = 60];
	optional string upNu = 4[(yd_fieldoptions.field_cname) = "参数二", (yd_fieldoptions.field_bufsize) = 60];
	optional string miaoshu = 5[(yd_fieldoptions.field_cname) = "描述", (yd_fieldoptions.field_bufsize) = 60];
	optional string miaoshuAdd = 6[(yd_fieldoptions.field_cname) = "附加描述", (yd_fieldoptions.field_bufsize) = 60];
	optional string tiaoJian = 7[(yd_fieldoptions.field_cname) = "达成条件", (yd_fieldoptions.field_bufsize) = 120];
	optional int64 boxId = 8[(yd_fieldoptions.field_cname) = "奖励ID"];
	optional int64 sdboxid = 9[(yd_fieldoptions.field_cname) = "困难奖励"];
}


message Sheet_operatejiangli
{
	repeated operatejiangli operatejiangli_List = 1[(yd_fieldoptions.field_arysize)=200];
}

message operatehoroscope
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "奖池id"];
	optional int64 daojuId = 2[(yd_fieldoptions.field_cname) = "道具id"];
	optional int32 daojuPrice = 3[(yd_fieldoptions.field_cname) = "道具单抽价格"];
	optional int32 cycle = 4[(yd_fieldoptions.field_cname) = "免费次数刷新时间"];
	optional string poolid = 5[(yd_fieldoptions.field_cname) = "奖池ID", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 wheelTime = 6[(yd_fieldoptions.field_cname) = "轮环时间(天)"];
	optional string ExchangeGroupID = 7[(yd_fieldoptions.field_cname) = "兑换组id", (yd_fieldoptions.field_bufsize) = 60];
	optional string boxid = 8[(yd_fieldoptions.field_cname) = "总奖池box", (yd_fieldoptions.field_bufsize) = 200];
	optional int64 luckid = 9[(yd_fieldoptions.field_cname) = "幸运奖池box"];
	optional string fateid = 10[(yd_fieldoptions.field_cname) = "累计幸运值奖励", (yd_fieldoptions.field_bufsize) = 60];
	optional string felicityid = 11[(yd_fieldoptions.field_cname) = "福气奖励", (yd_fieldoptions.field_bufsize) = 80];
}


message Sheet_operatehoroscope
{
	repeated operatehoroscope operatehoroscope_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message operatehoroscopePool
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int64 itemid = 2[(yd_fieldoptions.field_cname) = "物品"];
	optional int32 num = 3[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 prof = 4[(yd_fieldoptions.field_cname) = "职业"];
	optional int32 levelLimitA = 5[(yd_fieldoptions.field_cname) = "限制等级段下"];
	optional int32 levelLimitB = 6[(yd_fieldoptions.field_cname) = "限制等级段上"];
	optional int32 type = 7[(yd_fieldoptions.field_cname) = "奖池类型"];
	optional int32 typeParma = 8[(yd_fieldoptions.field_cname) = "奖池参数"];
	optional int32 probability = 9[(yd_fieldoptions.field_cname) = "掉落权重"];
	optional int64 groupID = 10[(yd_fieldoptions.field_cname) = "奖池组ID"];
	optional int32 order = 11[(yd_fieldoptions.field_cname) = "显示排序"];
}


message Sheet_operatehoroscopePool
{
	repeated operatehoroscopePool operatehoroscopePool_List = 1[(yd_fieldoptions.field_arysize)=1000];
}

message operatehoroscopeExchange
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 horoscopeId = 2[(yd_fieldoptions.field_cname) = "所属占星"];
	optional int64 prizeId = 3[(yd_fieldoptions.field_cname) = "奖励物品"];
	optional int32 prof = 4[(yd_fieldoptions.field_cname) = "职业"];
	optional int32 prizeNum = 5[(yd_fieldoptions.field_cname) = "奖励数量"];
	optional int32 luckNeed = 6[(yd_fieldoptions.field_cname) = "福气值数量"];
	optional int32 times = 7[(yd_fieldoptions.field_cname) = "限购次数"];
}


message Sheet_operatehoroscopeExchange
{
	repeated operatehoroscopeExchange operatehoroscopeExchange_List = 1[(yd_fieldoptions.field_arysize)=200];
}

message operateconst
{
	optional int64 constantid = 1[(yd_fieldoptions.field_cname) = "配置id"];
	optional int64 constantdata = 2[(yd_fieldoptions.field_cname) = "数据"];
	optional string stringdata = 3[(yd_fieldoptions.field_cname) = "字符串数据", (yd_fieldoptions.field_bufsize) = 180];
}


message Sheet_operateconst
{
	repeated operateconst operateconst_List = 1[(yd_fieldoptions.field_arysize)=100];
}

message operateunlimited
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "配置id"];
	optional int32 price = 2[(yd_fieldoptions.field_cname) = "价格"];
	optional string severPrice = 3[(yd_fieldoptions.field_cname) = "解锁计费点", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 itemId = 4[(yd_fieldoptions.field_cname) = "领取的物品ID"];
	optional int64 num = 5[(yd_fieldoptions.field_cname) = "单次领取的物品数量"];
}


message Sheet_operateunlimited
{
	repeated operateunlimited operateunlimited_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message operaterand
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 randtype = 2[(yd_fieldoptions.field_cname) = "冲榜类型"];
	optional int64 boxid = 3[(yd_fieldoptions.field_cname) = "奖励id"];
	repeated string canshu = 4[(yd_fieldoptions.field_cname) = "参数", (yd_fieldoptions.field_arysize) = 2, (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_operaterand
{
	repeated operaterand operaterand_List = 1[(yd_fieldoptions.field_arysize)=60];
}

message operatemooncard
{
	optional int32 dayNum = 1[(yd_fieldoptions.field_cname) = "天数"];
	optional int32 payBack = 2[(yd_fieldoptions.field_cname) = "返利紫钻"];
}


message Sheet_operatemooncard
{
	repeated operatemooncard operatemooncard_List = 1[(yd_fieldoptions.field_arysize)=60];
}

message operatefund
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 level = 2[(yd_fieldoptions.field_cname) = "达成等级"];
	optional int32 rebateS = 3[(yd_fieldoptions.field_cname) = "普通返还"];
	optional int32 rebateL = 4[(yd_fieldoptions.field_cname) = "超值返还"];
}


message Sheet_operatefund
{
	repeated operatefund operatefund_List = 1[(yd_fieldoptions.field_arysize)=60];
}

message operatetask
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "任务id"];
	optional int32 taskNan = 2[(yd_fieldoptions.field_cname) = "任务难度"];
	optional int32 taskType = 3[(yd_fieldoptions.field_cname) = "任务类型"];
	optional string num = 4[(yd_fieldoptions.field_cname) = "格式参数", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_operatetask
{
	repeated operatetask operatetask_List = 1[(yd_fieldoptions.field_arysize)=140];
}

message operateshop
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int64 itemid = 2[(yd_fieldoptions.field_cname) = "商品id"];
	optional int32 price = 3[(yd_fieldoptions.field_cname) = "原价"];
	optional int32 dPrice = 4[(yd_fieldoptions.field_cname) = "折扣价"];
	optional int32 moneyType = 5[(yd_fieldoptions.field_cname) = "货币类型"];
	optional int32 times = 6[(yd_fieldoptions.field_cname) = "限购次数"];
	optional int32 timeLimit = 7[(yd_fieldoptions.field_cname) = "时间h"];
}


message Sheet_operateshop
{
	repeated operateshop operateshop_List = 1[(yd_fieldoptions.field_arysize)=120];
}

message operatewish
{
	optional int32 level = 1[(yd_fieldoptions.field_cname) = "等级"];
	optional int64 exp = 2[(yd_fieldoptions.field_cname) = "经验祈愿"];
}


message Sheet_operatewish
{
	repeated operatewish operatewish_List = 1[(yd_fieldoptions.field_arysize)=2000];
}

message operateturntable
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int64 itemId = 2[(yd_fieldoptions.field_cname) = "物品ID"];
	optional int32 count = 3[(yd_fieldoptions.field_cname) = "物品数量"];
	optional int32 prof = 4[(yd_fieldoptions.field_cname) = "职业"];
	optional int32 prizeLevel = 5[(yd_fieldoptions.field_cname) = "奖品等级"];
	optional int32 weight = 6[(yd_fieldoptions.field_cname) = "权重"];
	optional int32 broadcast = 7[(yd_fieldoptions.field_cname) = "是否播放跑马灯"];
	optional int32 order = 8[(yd_fieldoptions.field_cname) = "显示排序"];
	optional int64 relation = 9[(yd_fieldoptions.field_cname) = "关联页签"];
}


message Sheet_operateturntable
{
	repeated operateturntable operateturntable_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message operateturntableTask
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "任务id"];
	optional int32 taskType = 2[(yd_fieldoptions.field_cname) = "任务类型"];
	optional int32 taskParam = 3[(yd_fieldoptions.field_cname) = "任务参数"];
	optional int64 itemId = 4[(yd_fieldoptions.field_cname) = "奖励物品id"];
	optional int32 itemCount = 5[(yd_fieldoptions.field_cname) = "奖励物品数量"];
}


message Sheet_operateturntableTask
{
	repeated operateturntableTask operateturntableTask_List = 1[(yd_fieldoptions.field_arysize)=20];
}
